.github/workflows/on_pull_request.yml

```diff
@@ -401,18 +471,26 @@ jobs:
       - name: Install Dependencies
         run: yarn install --immutable
 
-      - name: Download build artifacts
+      - name: Download website build artifact
         uses: actions/download-artifact@v2
         with:
-          name: compiled-js-and-types
-          path: packages/
+          name: compiled-website
+          path: packages/paste-website/
+          if-no-files-found: error
 
-      - name: Build website
-        run: yarn build:website
+      - name: Verify Cypress
         env:
-          AIRTABLE_APIKEY: ${{ secrets.AIRTABLE_APIKEY }}
-          AIRTABLE_BASEID: ${{ secrets.AIRTABLE_BASEID }}
-          GATSBY_DOCSEARCH_APIKEY: ${{ secrets.GATSBY_DOCSEARCH_APIKEY }}
+          # make sure every Cypress install prints minimal information
+          CI: 1
+        # print Cypress and OS info
+        run: |
```

Comment:
ℹ️  This will verify the cypress installation before running our tests, as well as print some logs about the environment the tests will run in. This can be helpful to ensure that we are using the system config that we expect.